tests/transform_dialect/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ iree_lit_test_suite(
   NAME
     lit
   SRCS
+    "conv_fill_spec_pad.mlir"
```

Comment:
This transform dialect conv 'test' should be disabled because of the Windows issue, is it still disabled?

Reply:
Now it uses iree-opt instead of iree-compile. I don't think it will cause the issue? @newling @erieaton-amd could you give it a try and see if breaks Windows?